configs/sst_container_tools.yaml

```diff
@@ -4,28 +4,38 @@ data:
   name: Container Tools
   description: >
     Container runtime and related packages maintained by the Container Tools SST
-  maintainer: sst_container_tools
+
+  maintainer: jnovy
```

Comment:
Why switch this to you instead of leaving it as the team?  The team assignment helps with other pieces.

Reply:
I'm happy to switch it back to sst_container_tools.